templates/shared/_card--masterclass.html

```diff
@@ -8,9 +8,11 @@
     {% endif %}
   </a>
   <div class="p-card__inner">
-    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted u-no-margin--bottom">Duration: {{ session.Duration }}</p>
+    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a>
+    </h3>
     <p>{{ session.Description }}</p>
+    <p class="u-text--muted u-no-margin--bottom">{{ session.Tags }}</p>
```

Comment:
Another idea to make this nicer: what about splitting it on the comma to have a list, and use chips for each tag?

Reply:
I think the muted text is better suited -- tags would be quite eye-catchey.

Reply:
Tags are meant to be so. And we already have muted text in the line below for the duration, so it's hard to distinguish them. Maybe ask input to a designer?

Reply:
@lorumic I had tried putting the chips in there, and @edlerd is right, it was taking a lot of space.
Check the screenshot:
![screenshot](https://github.com/canonical/masterclasses.canonical.com/assets/54525904/0d09405a-f53a-4004-8f2d-c7a0c2fcd487)

Reply:
This is screenshot with dense chips:
![screenshot 2](https://github.com/canonical/masterclasses.canonical.com/assets/54525904/7f8b63db-79d9-4995-a950-d9b2946fd5b7)

Reply:
I don't see anything wrong except that there should be more spacing between the chips and the duration line.

Reply:
After the discussion in our standups, we'll go ahead and use the dense chips.

Reply:
We've decided to go in favor of muted text for now, because chips are meant to be used interactively.